conda/mpich/meta.yaml

```diff
@@ -20,57 +18,35 @@ build:
 
 requirements:
   build:
-    - {{ moose_cc }}
-    - {{ moose_cxx }}
-    - {{ moose_fortran }}
-    - {{ moose_impl }}          # [linux]
-    - {{ moose_impl_xx }}       # [linux]
-    - {{ moose_libgcc }}        # [linux]
-    - {{ moose_libstd }}        # [linux]
-    - {{ moose_sysroot }}       # [linux]
-    - {{ moose_libcxx }}        # [osx]
-    - {{ moose_libclang }}      # [osx]
-    - {{ moose_openmp }}        # [osx]
-    - {{ moose_llvm_tools }}    # [osx]
-    - {{ moose_ld64 }}          # [osx]
+    - {{ base_mpich }}
+    - {{ base_mpicc }}
+    - {{ base_mpicxx }}
+    - {{ base_mpifort }}
     - autoconf                  # [unix]
     - automake                  # [unix]
     - libtool                   # [unix]
     - make                      # [unix]
     - gnuconfig                 # [arm64]
   host: []
   run:
-    - {{ moose_cc }}
-    - {{ moose_cxx }}
-    - {{ moose_fortran }}
-    - {{ moose_impl }}          # [linux]
-    - {{ moose_impl_xx }}       # [linux]
-    - {{ moose_libgcc }}        # [linux]
-    - {{ moose_libstd }}        # [linux]
-    - {{ moose_sysroot }}       # [linux]
-    - {{ moose_libcxx }}        # [osx]
-    - {{ moose_libclang }}      # [osx]
-    - {{ moose_openmp }}        # [osx]
-    - {{ moose_llvm_tools }}    # [osx]
-    - {{ moose_ld64 }}          # [osx]
+    - {{ base_mpich }}
+    - {{ base_mpicc }}
+    - {{ base_mpicxx }}
+    - {{ base_mpifort }}
     - autoconf                  # [osx]
     - automake                  # [osx]
     - libtool                   # [osx]
     - make                      # [osx]
     - {{ pin_compatible('setuptools', lower_bound='58', upper_bound='60') }}
     - mpi 1.0 mpich
 test:
-  script: run_test.sh
-  files:
-    - mpiexec.sh
-    - tests/helloworld.sh
+  commands:
+    - test -f $PREFIX/etc/conda/activate.d/activate_moose-mpich.sh
 
 about:
-  home: http://www.mpich.org/
-  license: MPICH
-  license_file: COPYRIGHT
-  license_family: Other
-  summary: 'A high performance widely portable implementation of the MPI standard.'
+  home: https://mooseframework.org/
+  license: LGPL 2.1
+  summary: 'Top Level packages designed to control environment variables for conda-forge mpich'
```

Comment:
I wrote this quickly in my PR. Can we fix the grammar here?

Reply:
Feel free to add a new commit, or you can post the diff and I can integrate it into my "address review" commit later.